src/transport/types.d.ts

```diff
@@ -1,3 +1,4 @@
+import BufferList from 'bl'
```

Comment:
So when you import `'bl'` but have `@types/bl` then it automagically pulls those definitions in rather than looking for the real `bl` (which isn't a dependency)?

Reply:
yeah, if you are in typescript it will look into `@types` namespace. You can also do this in jsdoc for specifying typedefs with import:

```
@typedef {import('bl')} BufferList
```